api/src/main/java/org/openmrs/module/idgen/SequentialIdentifierGenerator.java

```diff
@@ -88,7 +107,22 @@ public String getIdentifierForSeed(long seed) {
     	return identifier;
     }
 
-    //***** PROPERTY ACCESS *****
+	private String getPrefixProviderBeanNameIfApplicable(String prefix) {
+		if (StringUtils.isBlank(prefix)) {
+			return null;
+		}
+		if (prefix.startsWith(CONFIGURATION_PREFIX)) {
+			try {
+				return prefix.substring(prefix.indexOf(":") + 1).trim();
+			}
+			catch (IndexOutOfBoundsException e) {
+				throw new RuntimeException("Invalid Prefix Configuration. PrefixProvider bean name not provided", e);
```

Comment:
If you encounter any error while parsing the configuration you should _log_ a warning and behave as this was just a constant prefix.